packages/transport/test/core/backbone/CorrelationId.test.ts

```diff
@@ -0,0 +1,33 @@
+import { IDatabaseConnection } from "@js-soft/docdb-access-abstractions";
+import correlationIdLib from "correlation-id";
+import { AccountController } from "../../../src";
+import { RequestInterceptor } from "../../testHelpers/RequestInterceptor";
+import { TestUtil } from "../../testHelpers/TestUtil";
+
+describe("CorrelationId", function () {
+    let connection: IDatabaseConnection;
+    let testAccount: AccountController;
+    let interceptor: RequestInterceptor;
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+        const transport = TestUtil.createTransport(connection);
+        await transport.init();
+        const accounts = await TestUtil.provideAccounts(transport, 1);
+        testAccount = accounts[0];
+        interceptor = new RequestInterceptor((testAccount as any).synchronization.client);
+    });
+
+    afterAll(async function () {
+        await testAccount.close();
+        await connection.close();
+    });
+    test("should send correlation id to the backend when given", async function () {
```

Comment:
Writing "backbone" instead of "backend" as you do in the PR title should be more precise.